src/languages/cpp.js

```diff
@@ -73,7 +73,7 @@ function(hljs) {
       'atomic_bool atomic_char atomic_schar ' +
       'atomic_uchar atomic_short atomic_ushort atomic_int atomic_uint atomic_long atomic_ulong atomic_llong ' +
       'atomic_ullong new throw return ' +
-      'and or not',
+      'and or not __stdcall __try __finally',
```

Comment:
I think any __* is classified as a keyword, so you might need to update the keywords regex at the top of the file instead. Please also be sure to include a test.

Reply:
What do you mean? Turn me to a more specific place. I not found keywords regex.

Reply:
The point is that (according to https://en.cppreference.com/w/cpp/keyword):

> all identifiers that contain a double underscore __ in any position and each identifier that begins with an underscore followed by an uppercase letter is always reserved

Hence it's make sense to highlight all such identifiers as keywords rather than add them one by one.

Technically you'll need to add another submode for such keywords, see CPP_PRIMITIVE_TYPES for an example.

Reply:
@egor-rogov Thank you for the explanation, but unfortunately I do not know how to do what you said. If you can do this, thank you.